common/ethereum/src/eth_contracts/test_utils/mod.rs

```diff
@@ -54,6 +54,15 @@ pub fn get_sample_submission_material_with_weth_deposit() -> EthSubmissionMateri
     EthSubmissionMaterial::from_str(&get_sample_submission_material_string_n("eth", 2).unwrap()).unwrap()
 }
 
+const PTOKENS_ROUTER_METADATA_EVENT_RECEIPT: &str = "src/eth_contracts/test_utils/ptokens-router-metadata-event-1.json";
+
+pub fn get_ptokens_router_metadata_event_receipt() -> Result<EthReceipt> {
+    match Path::new(PTOKENS_ROUTER_METADATA_EVENT_RECEIPT).exists() {
+        true => EthReceipt::from_str(&read_to_string(PTOKENS_ROUTER_METADATA_EVENT_RECEIPT)?),
+        false => Err(format!("✘ Cannot find {} file!", PTOKENS_ROUTER_METADATA_EVENT_RECEIPT).into()),
+    }
```

Comment:
I'm surprised this isn't coming up in the linter since it normally complains about matching on bools. I however I'm not fussed, and this is a test util fxn to 🤷🏻